demo-client/src/main/java/com/illia/client/service/FileHandlingService.java

```diff
@@ -0,0 +1,48 @@
+package com.illia.client.service;
+
+import com.illia.client.config.ClientConfig;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.ByteArrayInputStream;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+@Service
+public class FileHandlingService {
+
+    @Autowired
+    ClientConfig clientConfig;
+
+    // needed?
```

Comment:
remove if not immediately needed, you can still find it in git commit history if needed later